.github/workflows/push.yaml

```diff
@@ -7,12 +7,16 @@ on:
   pull_request:
 
 permissions:
-  contents: read
+  contents: none
+  id-token: none
```

Comment:
Nit: for consistency I tend to do:

```
permissions: {}
```

as recommended by Zizmor

Reply:
this is not used, I need to remove it 👍